.github/workflows/github-release-build-and-deploy.yml

```diff
@@ -48,7 +50,7 @@ jobs:
           cat <(echo -e "${{ secrets.OSSRH_GPG_SECRET_KEY }}") | gpg --batch --import
           gpg --list-secret-keys --keyid-format LONG
       - name: Publish package
-        run: mvn --batch-mode deploy
+        run: mvn --batch-mode deploy -DskipITs=true
```

Comment:
@abecerra According to maven docs, this should only be `-DskipITs` (so without the `=true`) https://maven.apache.org/surefire/maven-failsafe-plugin/examples/skipping-tests.html

Reply:
According to quarkus guide (below the code block, in point 1), the skipITs must have the true value.
I guess we can test with the true value first and if it does not work, remove it.

https://quarkus.io/guides/maven-tooling#build-tool-maven